src/elastic_search/mod.rs

```diff
@@ -8,7 +8,7 @@ use testcontainers::{
 const NAME: &str = "docker.elastic.co/elasticsearch/elasticsearch";
 const TAG: &str = "7.16.1";
 
-#[derive(Debug, Default)]
+#[derive(Debug, Default, Clone)]
 pub struct ElasticSearch {
     _priv: (),
```

Comment:
One thing that I found interesting is this pattern vs (for example) `pub struct DynamoDb;`

Out of curiosity:
Is there a reason for having this unused member variable? ^^

Reply:
>  Is there a reason for having this unused member variable? ^^

Yes, to provide encapsulation, this allows the image to be extended without breaking future changes.

`pub struct DynamoDb;` allows you to create an instance simply by using `DynamoDb`. Therefore adding a new field will break this usage.
Although `ElasticSearch` requires the use of `new()` or `default()` so that we can safely add new fields.

However, some images are still missing this pattern and should be updated in the future